node/pkg/governor/mainnet_tokens_test.go

```diff
@@ -64,25 +64,6 @@ func TestTokenListTokenAddressDuplicates(t *testing.T) {
 	}
 }
 
-func TestTokenListDecimalRange(t *testing.T) {
-	tokenConfigEntries := tokenList()
-
-	/* Assume that all governed token entries will have decimals of 6 or 8 */
-	for _, tokenConfigEntry := range tokenConfigEntries {
-		d := tokenConfigEntry.decimals
-		assert.Condition(t, func() bool { return d == 6 || d == 8 })
-	}
-}
-
-func TestTokenListEmptySymbols(t *testing.T) {
-	tokenConfigEntries := tokenList()
-
-	/* Assume that all governed token entry symbol strings will be greater than zero */
-	for _, tokenConfigEntry := range tokenConfigEntries {
-		assert.Greater(t, len(tokenConfigEntry.symbol), 0)
-	}
-}
```

Comment:
reading above, the exception case is Solana tokens, could we instead in the loop just continue for Solana and still get said protection for the other chains where we expect to symbols to be non-zero or were there other unexpected cases on other chains?

Reply:
Put the test back and just have it skip Solana.